src/mk.sh

```diff
@@ -401,6 +401,10 @@ function kernel_deploy
   local runtime=0
   local ret=0
 
+  if [[ $(id -u) == 0 ]]; then
+    warning "kw deploy should not be run as root"
```

Comment:
Instead of use `warning`, use `complain`. Also, we probably want to exit with `EPERM` (1) or similar.

Additionally, I think it makes more sense to add it under the local deploy part (LOCAL_TARGET).

Reply:
Yes, it will be better under local deploy. I'll do the changes.